src/server/verification/verificationAPI.js

```diff
@@ -39,69 +35,41 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
-      }
       let result = { ok: 0 }
-      if (!user.isVerified && !conf.skipFaceRecognition)
-        result = await verifier
-          .verifyUser(user, verificationData)
-          .catch(e => {
-            log.error('Facerecognition error:', { e })
-            GunDBPublic.gun.get(sessionId).put({ isNotDuplicate: false, isLive: false, isError: e.message })
-            return { ok: 1, error: e.message, isVerified: false }
-          })
-          .finally(() => {
-            //cleanup
-            log.info('cleaning up facerecognition files')
-            fsPromises.unlink(verificationData.facemapFile)
-            fsPromises.unlink(verificationData.auditTrailImageFile)
-          })
-      else {
-        GunDBPublic.gun.get(sessionId).put({ isNotDuplicate: true, isLive: true, isEnrolled: true }) // publish to subscribers
+
+      if (user.identifier === undefined || sessionId === undefined || images === undefined) {
+        return res.status(400).send({ ok: 0, error: 'invalid input', isVerified: false })
+      }
+      const sessionRef = GunDBPublic.session(sessionId)
+
+      sessionRef.put({ isStarted: true })
+
+      log.debug('written FR status to gun', { data: await sessionRef.get() })
+
+      const imagesBase64 = images.map(img => img.base64)
+
+      if (!user.isVerified && !conf.skipFaceRecognition) {
+        try {
+          result = await Verifications.verifyUser(user, sessionId, imagesBase64, storage)
+        } catch (e) {
+          log.error('Facerecognition error:', { e })
+          sessionRef.put({ isDuplicate: true, isLive: false, isError: e.message })
+          result = { ok: 1, error: e.message, isVerified: false }
+        }
+      } else {
+        sessionRef.put({ isDuplicate: false, isLive: true, isEnrolled: true }) // publish to subscribers
         // mocked result for verified user or development mode
         result = {
           ok: 1,
           isVerified: true,
-          enrollResult: { alreadyEnrolled: true, enrollmentIdentifier: verificationData.enrollmentIdentifier }
-        } // skip facereco only in dev mode
-      }
-      log.debug('Facerecogintion result:', result)
-      if (result.isVerified) {
-        log.debug('Whitelisting new user', user)
-        await Promise.all([
-          AdminWallet.whitelistUser(user.gdAddress, user.profilePublickey),
-          storage
-            .updateUser({ identifier: user.loggedInAs, isVerified: true })
-            .then(updatedUser => log.debug('updatedUser:', updatedUser))
-        ])
-          .then(
-            _ =>
-              GunDBPublic.gun
-                .get(sessionId)
-                .get('isWhitelisted')
-                .put(true) // publish to subscribers
-          )
-          .catch(e => {
-            log.error('Whitelisting failed', e)
-            GunDBPublic.gun.get(sessionId).put({ isWhitelisted: false, isError: e.message })
-          })
+          enrollResult: { alreadyEnrolled: true, enrollmentIdentifier: true }
```

Comment:
not sure these are used anymore, or returned from the new backend

Reply:
removed **enrollResult**  it is not used